lib/components/play-pause-button.js

```diff
@@ -16,10 +17,10 @@ class PlayPauseButton extends React.Component {
   render() {
     var style = {
       padding: '0.5em',
-      marginBottom: 0,
       backgroundColor: '#C36351',
-      borderRadius: '10',
+      borderRadius: '1em',
```

Comment:
I do think we could turn the rounded corner down a bunch.  maybe to 3 or 4 px - or 0.25em